slices/bind9-libs.yaml

```diff
@@ -0,0 +1,31 @@
+package: bind9-libs
+
+essential:
+  - bind9-libs_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgssapi-krb5-2_libs
+      - libjson-c5_libs
+      - libkrb5-3_libs
+      - liblmdb0_libs
+      - libmaxminddb0_libs
+      - libnghttp2-14_libs
+      - libssl3t64_libs
+      - libuv1t64_libs
+      - libxml2_libs
+      - zlib1g_libs
+    contents:
+      /usr/lib/*-linux-*/libbind9-*.so:
+      /usr/lib/*-linux-*/libdns-*.so:
+      /usr/lib/*-linux-*/libirs-*.so:
+      /usr/lib/*-linux-*/libisc-*.so:
+      /usr/lib/*-linux-*/libisccc-*.so:
+      /usr/lib/*-linux-*/libisccfg-*.so:
+      /usr/lib/*-linux-*/libns-*.so:
+  
```

Comment:
There appears to be a trailing space character at the end of this blank line, can you remedy?